### PR TITLE
[DOCS-5651] Attachment properties for email connector

### PR DESCRIPTION
**Pull request by m-hulbert**

### CODE REVIEW

File: docs/modeling/connectors/ootb/email.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ The following are the parameters that can be passed to the email connector as in
 | `text` | A plain text email body | String | No |
 | `html` | An HTML email body | String | No |
 | `charset` | Define the *charset* of the email | String | No | 
+| `nodeId` | The node ID of the file to attach to the email | String | No |
+| `uri` | The URI of the file to attach to the email | String | No |
+| `files` | A [file](../../modeling/files.md) uploaded in a process and set as a process variable or uploaded as part of a form or another connector to attach to the email | File | No |
```
Comment (nazarethjim) [inline docs/modeling/connectors/ootb/email.md]:
```suggestion
| `files` | A [file](../../files.md) uploaded in a process and set as a process variable or uploaded as part of a form or another connector to attach to the email | File | No |
```

**Review by nazarethjim — CHANGES_REQUESTED** [review]

File: docs/modeling/connectors/ootb/email.md (hunk 1 above)
Comment (pmartinezga) [reply docs/modeling/connectors/ootb/email.md]: Yes, that is the only thing I spotted here

**Review by pmartinezga — CHANGES_REQUESTED** [review]
There is only a relative path that is not correct.

File: docs/modeling/connectors/ootb/email.md (hunk 1 above)
Comment (m-hulbert) [reply docs/modeling/connectors/ootb/email.md]: Thank you for the good spot @nazarethjim !

**Pull request merged**